src/bp_tree_node.erl

```diff
@@ -346,9 +347,23 @@ rotate_left(LNode, ParentKey, RNode, Order) ->
 %%--------------------------------------------------------------------
 -spec fold(bp_tree:fold_start_spec(), bp_tree:tree_node(),
     bp_tree:fold_fun(), bp_tree:fold_acc()) -> bp_tree:fold_acc().
-fold(KeyOrPos, #bp_tree_node{children = LChildren}, Fun, Acc) ->
+fold(KeyOrPos, #bp_tree_node{leaf = true, children = LChildren}, Fun, Acc) ->
```

Comment:
jak leaf może mieć dzieci?

Reply:
Dać krótki komentarz wyjaśniający to na górze modułu.